2-mandatory/4-restaurant.js

```diff
@@ -19,9 +19,20 @@ const MENU = {
   falafel: 7.25,
 };
 
+const order = (price) => (balance) =>
```

Comment:
Very good - functions that produce functions is an important mental shift. That's a very powerful technique. Don't overuse it though :)